### DISCUSSION TITLE
[Feature Request] Master password

### DISCUSSION THREAD

**Discussion by proudmuslim-dev** [diegogurpegui/nos2x-fox#24] [Ideas]
I don't really feel comfortable keeping my private key unencrypted on my machine- as of right now the only NIP-07 extensions with this feature are BTC wallets, and I have no desire to use BTC/LN

I don't think I'm alone in wanting this. I'd PR the feature myself, but I unfortunately I've never worked with JS or React

**Comment by diegogurpegui** [comment]
So you are thinking of a master password that you would have to enter each time you open the extension? or each time you sign?

I think this is related to #13

**Comment by diegogurpegui** [comment]
Solved with feature "PIN Protection"
https://github.com/diegogurpegui/nos2x-fox/releases/tag/v1.18.0
